front/app/components/IdeaCards/shared/Filters/SortingBox/SortTypeButton.tsx

```diff
@@ -1,20 +1,52 @@
-import React, { useState } from 'react';
+import React from 'react';
 
-import { Box, Text, Icon, colors } from '@citizenlab/cl2-component-library';
-import { useTheme } from 'styled-components';
+import { Box, BoxProps, colors, Icon } from '@citizenlab/cl2-component-library';
+import { hideVisually, darken } from 'polished';
+import styled, { useTheme } from 'styled-components';
 
-import { FrontOfficeSortOptions } from 'api/ideas/types';
+import { IdeaDefaultSortMethod } from 'api/phases/types';
 
 import { useIntl } from 'utils/cl-intl';
 
-import {
-  getButtonBackgroundColor,
-  getIconNameForSortingOption,
-  getLabelForSortingOption,
-} from './utils';
+import { getIconNameForSortingOption, getLabelForSortingOption } from './utils';
+
+const HiddenRadio = styled.input.attrs({ type: 'radio' })`
+  ${hideVisually()};
+`;
+
+const Label = styled.label<{ selected: boolean }>`
+  display: flex;
+  align-items: center;
+  padding: 8px;
+  cursor: pointer;
+  border-radius: ${({ theme }) => theme.borderRadius};
+  width: 100%;
+  font-size: ${({ theme }) => theme.fontSizes.base}px;
+
+  ${({ theme: { colors }, selected }) =>
+    selected
+      ? `
+    background: ${colors.tenantPrimary};
+    color: ${colors.white};
+    
+    &:hover {
+      background: ${darken(0.15, colors.tenantPrimary)};
+    } `
+      : `
+    background: ${colors.white};
+    color: ${colors.textPrimary};
+    
+    &:hover {
+      background: rgba(132, 147, 158, 0.15);
```

Comment:
Can we use a colour from the "color" in the component library instead for background? I think I used colors.grey200 or something originally :) We can update the other filters to use the same.